[2024-05-04T12:35:45.669+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-04T12:35:45.700+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: for_project.get_files scheduled__2024-05-04T11:00:00+00:00 [queued]>
[2024-05-04T12:35:45.713+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: for_project.get_files scheduled__2024-05-04T11:00:00+00:00 [queued]>
[2024-05-04T12:35:45.714+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-04T12:35:45.774+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): get_files> on 2024-05-04 11:00:00+00:00
[2024-05-04T12:35:45.827+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'for_project', 'get_files', 'scheduled__2024-05-04T11:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/dag_for_project.py', '--cfg-path', '/tmp/tmp6_z9lp5t']
[2024-05-04T12:35:45.830+0000] {standard_task_runner.py:91} INFO - Job 36: Subtask get_files
[2024-05-04T12:35:45.850+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=88148) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-04T12:35:45.852+0000] {standard_task_runner.py:63} INFO - Started process 88208 to run task
[2024-05-04T12:35:46.008+0000] {task_command.py:426} INFO - Running <TaskInstance: for_project.get_files scheduled__2024-05-04T11:00:00+00:00 [running]> on host 5bc5768c5236
[2024-05-04T12:35:46.505+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='for_project' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-05-04T11:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-04T11:00:00+00:00'
[2024-05-04T12:35:46.523+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-04T12:35:47.780+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-04T12:35:51.469+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-04T12:35:51.591+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-04T12:35:51.591+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-04T12:35:51.599+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=for_project, task_id=get_files, execution_date=20240504T110000, start_date=20240504T123545, end_date=20240504T123551
[2024-05-04T12:35:51.661+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-04T12:35:51.689+0000] {taskinstance.py:3482} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-05-04T12:35:51.701+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
